### DISCUSSION TITLE
How to disable ESLint in development mode?

### DISCUSSION THREAD

**Discussion by piliugin-anton** [nuxt/nuxt#9799] [Questions (Nuxt 2)]
Hello. I need to disable ESLint in development mode, how could I do it?

**Comment by dmitryburov** [comment]
You can just remove the eslint dev-dependencies and scripts in package.json.

Or modify config and rules in `.eslintignore`,  `.eslintrc.js`.
Also, as variant, create build rule in `nuxt.config.js`

 e.g.
```js
build: {
    extend (config, { isDev, isServer, isClient }) {
      if (!isDev) {
        config.module.rules.push({
          enforce: 'pre',
          test: /\.(js|vue)$/,
          loader: 'eslint-loader',
          exclude: /(node_modules)/
        })
      }
    }
}
```
